<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SunnyTcsGraphicTestClass</class>
 <widget class="QMainWindow" name="SunnyTcsGraphicTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SunnyTcsGraphicTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>craft</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QPushButton" name="deleteBtn">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>创建</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="linkRBtn">
               <property name="text">
                <string>连接线</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="routeRBtn">
               <property name="text">
                <string>子订单</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="moveRBtn">
               <property name="text">
                <string>移动</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="SunnyTcsCraftGraphicsView" name="graphicsView_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>map</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>创建</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="map_pt_crt">
               <property name="text">
                <string>点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="map_ph_crt">
               <property name="text">
                <string>直线</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="map_aph_crt">
               <property name="text">
                <string>曲线</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="map_ve_crt">
               <property name="text">
                <string>车</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>移动</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="map_edit_rbtn">
            <property name="text">
             <string>编辑模式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="distr_init_btn">
            <property name="text">
             <string>初始化分布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="map_del_btn">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="x_mirror_btn">
            <property name="text">
             <string>X镜像</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="y_mirror_btn">
            <property name="text">
             <string>y镜像</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotate_btn">
            <property name="text">
             <string>旋转</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="SunnyTcsCraftGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SunnyTcsCraftGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>SunnyTcsCraftGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SunnyTcsGraphicTest.qrc"/>
 </resources>
 <connections/>
</ui>
